python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -380,16 +385,17 @@ def end_element(name):
             # is a root object)
             if name == 'class':
                 obj = self.stack[-1]
-                for attr, item in obj._schema.allItems():
+                def schema_assign(_tuple, obj):
+                    attr = _tuple[0]
+                    item = _tuple[1]
                     if not attr in obj._data:
                         #logger.info("Opening: %s" % attr)
                         if item._meta["sequence"] == 1:
-                            obj.setSchemaAttribute(attr, makeGangaListByRef(obj._schema.getDefaultValue(attr)))
-                            #setattr(obj, attr, makeGangaListByRef(obj._schema.getDefaultValue(attr)))
+                            obj.setSchemaAttribute(attr, makeGangaListByRef(obj._schema.getDefaultValue(attr, make_copy=False)))
                         else:
-                            obj.setSchemaAttribute(attr, obj._schema.getDefaultValue(attr))
-                            #setattr(obj, attr, obj._schema.getDefaultValue(attr))
-
+                            obj.setSchemaAttribute(attr, obj._schema.getDefaultValue(attr, make_copy=False))
+                for item in obj._schema.allItems():
+                    schema_assign(item, obj=obj)
```

Comment:
This was a used in a map and the scope was clearer but I moved back and this is the result.
I'll look into reverting this.